lib/ruby_lsp/addon.rb

```diff
@@ -99,8 +99,10 @@ def formatted_errors
     end
 
     sig { returns(String) }
-    def backtraces
-      @errors.filter_map(&:backtrace).join("\n\n")
+    def errors_details
+      @errors.map do |error|
+        T.must(error.class.name) + ": " + error.message + "\n" + Array(error.backtrace).join("\n")
+      end.join("\n\n")
```

Comment:
I think this would include everything
```suggestion
      @errors.map do |error|
        error.full_message
      end.join("\n\n")
```